backend/baseclass.pm

```diff
@@ -25,10 +25,12 @@ our $backend;
 use parent qw(Class::Accessor::Fast);
 __PACKAGE__->mk_accessors(
     qw(
-      update_request_interval last_update_request
-      screenshot_interval last_screenshot last_image
-      reference_screenshot)
-);
+      update_request_interval last_update_request screenshot_interval
+      last_screenshot last_screenshot_name_ last_image
```

Comment:
because I need a property to store the attribute. And then I need a function so the testapi can fetch it. But as the testapi<->backend communication are hashes, so I can't call the attribute function directly.